docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -705,47 +705,38 @@ Once clicked, a File Explorer will open, and you can select a name and location
+.. note::
+    You need to run IQ Control Center as an administrator on Windows in order to import resource files.
 
-4. Inside of the Resources folder you’ll find Defaults, Firmware, and Tabs
-5. If you have been given a resource file with a number.json (42.json for example), copy the file into the Firmware folder
-6. If you have been given files with the names advanced, general, testing, or tuning, copy the files into the corresponding folder inside of the Tabs folder
+3. Once the application runs, navigate to the Menu in the top left and select 'Import Resource Pack'.
 
-Linux (Ubuntu)
-------------------
-1. Open a new terminal window, and navigate to the IQ Control Center installation. By default this is Home/~user~/IQ Control Center
-2. Inside you will see the following
+.. image:: ../_static/control_center_pics/windows/windows_menu_import_resource_pack.png
 
-.. image:: ../_static/control_center_pics/linux/linux_install_folder.png
+4. A popup will appear requesting that you run Control Center as an administrator. Since you already did this in step 1, click 'OK' and proceed to the next step.
 
-3. Enter IQ Control Center, and inside you will find a folder called Resources
-4. Enter the Resources folder, and inside you will see Defaults, Firmware, and Tabs
-5. If you have been given a resource file with a number.json (42.json for example), copy the file into the Firmware folder
-6. If you have been given files with the names advanced, general, testing, or tuning, copy the files into the corresponding folder inside of the Tabs folder
+.. image:: ../_static/control_center_pics/windows/windows_administrator_privileges_required_popup.png
 
-Mac
-------
-1. Open a new terminal window, and navigate to the IQ Control Center installation. By default this is Applications/IQ Control Center
-2. Enter the IQ Control Center folder. Inside you will see the following
+5. The File Explorer will appear. Navigate to the .zip file that was provided. In this example, the .zip file is named ResourcePackExample and it is located in Downloads. Select the .zip file and click 'Open'.
+
+.. image:: ../_static/control_center_pics/windows/windows_resource_pack_file_explorer.png
 
-.. image:: ../_static/control_center_pics/mac/mac_in_the_install.png
+6. After the .zip file is imported, a popup will appear requesting that you restart Control Center. Click 'OK' and restart Control Center, but this time you do not have to run as an administrator.
 
-3. Enter IQ Control Center.app/Contents/MacOS. Inside you will see the following
+.. image:: ../_static/control_center_pics/windows/windows_application_restart_required_popup.png
 
-.. image:: ../_static/control_center_pics/mac/mac_macos_folder.png
+7. If the provided resource .zip file includes default files, you will find them in the dropdown menu in the General tab next to MODULE PRE-SETTINGS. The other resource files will include additional configuration options based on your connected module.
```

Comment:
Could we link "defaults files" up to the Configuration with Defaults section above just in case people don't know what "default files" means?